code/modules/admin/IsBanned.dm

```diff
@@ -1,6 +1,10 @@
 /world/IsBanned(key, address, computer_id, type)
 	var/real_login = type != "goonchat" //Certain actions don't make sense to perform for the cookie checks.
 
+	// Some "world-type" bans relegated to BYOND's internal cuisine. That's how TG does it.
+	if (type == "world")
+		return ..()
```

Comment:
What does this do?

Reply:
This defers the banned check to byond's internal proc for some kind of connections. In this case, it's for when you've been redirected through World.Topic().

```dm
	if (type == "world")
		return ..() //shunt world topic banchecks to purely to byond's internal ban system
```

TG seems to do it just fine.